s/sentry-python/sentry-python_ubi_8.4.sh

```diff
@@ -28,11 +28,11 @@ fi
 
 # Dependency installation
 dnf update
```

Comment:
Is this `dnf update` really necessary? If yes, please add `-y` to the command so that it does not get blocked for user input, otherwise please remove it.

Reply:
dnf update not necessary. Hence removed.